config.m4

```diff
@@ -390,6 +415,62 @@ if test "$PHP_PCRE_DIR" != "no"; then
   fi
 fi
 
+PHP_ARG_WITH(mongodb-openssl, for OpenSSL support,
+[  --with-mongodb-openssl[=DIR]     mongodb: Include OpenSSL support], auto, no)
```

Comment:
The extension already had openssl support without this.
And it was configurable through --with-openssl-dir. This doesn't look right at all